modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -42,9 +42,8 @@ registry to use storage.
 ifndef::ibm-z,ibm-power[bare metal.]
 ifdef::ibm-z[IBM Z.]
 ifdef::ibm-power[IBM Power.]
-* Persistent storage provisioned for your cluster, such as
-ifndef::ibm-z[Red Hat OpenShift Container Storage.]
-ifdef::ibm-z[NFS.]
+ifndef::ibm-z[* Persistent storage provisioned for your cluster, such as Red Hat OpenShift Container Storage.]
+ifdef::ibm-z[* Persistent storage provisioned for your cluster.]
```

Comment:
This is a follow-up fix to https://github.com/openshift/openshift-docs/pull/26512.